### ISSUE TITLE
[Bug]: Alarm notifications

### ISSUE THREAD

**Issue by Tehnikis** [bvis/aegis-hass#317]
### Describe the bug

At the moment, my Home Assistant does not receive alarm events from Ajax. For example, when Zone 3 in Area 2 is triggered, including the zone name, nothing happens in Home Assistant notifications. An alarm event has never been reflected in Home Assistant notifications.

### Steps to reproduce

.

### Expected behavior

It would be very convenient to receive push notifications from Home Assistant for specific Ajax alarm events.

### Home Assistant version

2026.06.02

### Integration version

1.7.0-beta.8

### Installation method

HACS

### App label (co-branded app)

Ajax

### FCM push notifications configured?

Yes

### Relevant logs

```text

```

**Comment by bvis** [comment]
Hi @Tehnikis — alarm events are already exposed via an Event entity, which you can use to send Home Assistant push notifications through a normal automation.

You're running 1.7.0-beta.8, which is quite old. Could you:
1. Update to the latest release (1.12.1) via HACS,
2. Check that the alarm panel and event entities appear,
3. If notifications still don't fire, share a debug log so I can take a look?

Thanks!

**Comment by Tehnikis** [comment]
*Hello,*

Thank you for your reply.

Please find the attached screenshots. I am not receiving any notifications.
I keep updating the integration regularly, but the issue remains.

Thank you.

**Comment by bvis** [comment]
Following up: I don't see any screenshots or logs in the report, and without them there's nothing I can analyze.

Could you enable debug logging and capture what happens when an alarm triggers? To do it, add this to your `configuration.yaml`, restart Home Assistant, reproduce the alarm, and then attach the log here:

```yaml
logger:
  default: info
  logs:
    custom_components.aegis_ajax: debug
```

A diagnostics dump also helps (Settings → Devices & Services → Aegis for Ajax → ⋮ → Download diagnostics). Thanks!

**Comment by bvis** [comment]
Hi @Tehnikis — since there's been no further response, I'll assume alarm events/notifications are now working for you on the current release, so I'm closing this as resolved.

If it turns out something still isn't behaving as expected, please open a new issue attaching the debug log and diagnostics dump requested above so I have something concrete to analyze. Thanks!
